### PR TITLE
add optimization for OPPO.

### PR DESCRIPTION
**Pull request by xianyinchen**
提交数据：
render_pid：渲染进程
buffer_size：渲染表面大小
shader_compile：着色器片段编译（1开始，0关闭）
load_scene：加载场景（1开始，0关闭）

### CODE REVIEW

**Comment by minggo** [comment]
Should update the CMakeLists.txt too, because now support cmake too.

File: cocos/platform/CCDataManager.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+Copyright (c) 2010-2012 cocos2d-x.org
+Copyright (c) 2013-2016 Chukong Technologies Inc.
+Copyright (c) 2017-2018 Xiamen Yaji Software Co., Ltd.
```
Comment (minggo) [inline cocos/platform/CCDataManager.h:4]: Copyright (c) 2019 Xiamen Yaji Software Co., Ltd.

**Comment by huachangmiao** [comment]
What optimization did you do?

**Comment by minggo** [comment]
Report some events, such as scene loading, shader compiling, to device. Then device may provide more resources to the application.

**Comment by changong** [comment]
@minggo OPPO R9s can not get more resources. Which OPPO device has been tested?

**Comment by minggo** [comment]
@changong i talked with OPPO developers, they said currently only `reno` had effect. They will make more devices take effect in future.

**Comment by changong** [comment]
> @changong i talked with OPPO developers, they said currently only `reno` had effect. They will make more devices take effect in future.

Thank you.

**Comment by changong** [comment]
It has no effect on OPPO Reno(PCAM00).

**Comment by minggo** [comment]
@changong now it only has effect at scene loading and compiling shader. But you can invoke the API yourself if you like in your codes to get more power.

**Pull request merged**